reana-ui/src/pages/privacyNotice/PrivacyNotice.js

```diff
@@ -26,7 +26,7 @@ export default function PrivacyNoticePage() {
 function PrivacyNotice() {
   return (
     <Container className={styles["container"]}>
-      <Title as="h2">CERN Privacy Notice PN00915</Title>
+      <Title as="h2">CERN privacy notice PN00915</Title>
```

Comment:
Which one should I choose?

- _CERN privacy notice PN00915_ (sentence case, only first word capitalised)
- _CERN Privacy notice PN00915_ (to be consistent with _Privacy notice_ in the footer)
- _CERN Privacy Notice PN00915_ (because PN is the acronym of Privacy Notice)

Reply:
I guess we can go for the 3rd choice, since as you mentioned the AmE capitalisation has the relation to the acronym, and since there are added words "CERN" and "PN00915" which make the connection to the BrE capitalisation in the footer text a  somewhat loose one. BTW in [CERN service pages](https://cern.service-now.com/service-portal?id=privacy_policy&se=open-science-infrastructure&notice=reana), the text reads "Privacy Notice: REANA (PN00915)".

Another 4th solution would be to use the page title "Privacy notice", so that we would have 100% correspondence between the page title and the advertised link title in the page footer; that would be great. And we would mention the CERN PN number only somewhere in the page body, perhaps at the bottom around  the text "Last revision: 2020-10-09 18:04:55" we could add a link to "CERN Privacy Notice: REANA (PN00915)"?

Reply:
Updated the PR with the 4th solution